action.yml

```diff
@@ -18,6 +18,10 @@ inputs:
     description: 'Optional additional arguments to `tailscale up`'
     required: false
     default: ''
+  daemon_args:
```

Comment:
Recommend using dash, instead of underscore, to match GitHub's own actions like https://github.com/actions/checkout/blob/main/action.yml

Reply:
https://github.com/tailscale/github-action/pull/64/commits/bc0604ed9c0789ca29f0bffc10c4e817a314b477